sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/rest_client.hpp

```diff
@@ -31,7 +31,7 @@ namespace Azure { namespace Storage { namespace Blobs {
     /**
      * The version used for the operations to Azure storage services.
      */
-    constexpr static const char* ApiVersion = "2020-02-10";
+    constexpr static const char* ApiVersion = "2020-08-04";
```

Comment:
Should we mention in Changelog that SDK client will be now talking to a newest API version?

Reply:
Customers should be agnostic about API version. They only care about which features are available and which are not.

Reply:
I would suggest the API version is added to changelog. It makes answering the question "Since which SDK version is REST version XXXX-XX-XX used?" a lot easier.